lib/utils/couchstore/src/test/test.js

```diff
@@ -0,0 +1,142 @@
+'use strict';
+const session = require('express-session');
+const couchStore = require('..')(session);
+const dbclient = require('abacus-dbclient');
+const moment = require('abacus-moment');
+const server = () => process.env.DB;
+
+describe('abacus-couchStore', () => {
+  let store;
+  let dbName = 'abacus-test-dashboard';
+  let cookie = { name: 'test-cookie', cookie: { maxAge: 5000 } };
+
+  before(() => {
```

Comment:
You need to add a logic similar to this one: https://github.com/cloudfoundry-incubator/cf-abacus/blob/62578a4dc92030fe83ac6ae6249ccbb53cbc314d/lib/utils/couchclient/src/test/test.js#L39-L49

This will prevent the tests from running with `mongoclient` or any future clients.